blurring_as_a_service/inference_pipeline/submit_inference_pipeline.py

```diff
@@ -10,58 +12,91 @@
 
 
 @pipeline()
-def inference_pipeline(mounted_root_folder, relative_paths_files_to_blur, model):
-    outputs = BlurringAsAServiceSettings.get_settings()["inference_pipeline"]["outputs"]
+def inference_pipeline():
+    # Iterate over all customers
+    for customer in inference_settings['customers']:
+        customer_name = customer['name']
 
-    detect_and_blur_sensitive_data_step = detect_and_blur_sensitive_data(
-        mounted_root_folder=mounted_root_folder,
-        relative_paths_files_to_blur=relative_paths_files_to_blur,
-        model=model,
-    )
-    detect_and_blur_sensitive_data_step.outputs.results_path = Output(
-        type="uri_folder", mode="rw_mount", path=outputs["results_path"]
-    )
-    detect_and_blur_sensitive_data_step.outputs.yolo_yaml_path = Output(
-        type="uri_folder", mode="rw_mount", path=outputs["yolo_yaml_path"]
-    )
+        container_root_formatted = customer['container_root'].format(
```

Comment:
what does this variable mean? can you explain in a comment what this means or use another variable name?